src/lex/token.rs

```diff
@@ -11,14 +10,18 @@ pub enum Token {
 	Percent,
 	Caret,
 	Ampersand,
-	Space,
-	Newline,
-	String(String),
-	Tab,
+	Identifier(String),
```

Comment:
are you not gonna lex keywords?

Reply:
won't keywords just be a part of the enum as values